src/middlewared/middlewared/plugins/system.py

```diff
@@ -569,6 +569,10 @@ def __init__(self, *args, **kwargs):
 
         return data
 
+    @accepts()
+    def https_protocols_choices(self):
```

Comment:
`ui_httpsprotocols_choices`

It should also return a dict since thats the default we use for choices every where else (or almost).

Also needs a docstring